Add TFA.me integration to Home Assistant (Help page) (#40974)
home-assistant/home-assistant.github.io: source/_integrations/a_tfa_me_1.markdown

| [A4](https://www.tfa-dostmann.de/en/product/professional-temperature-humidity-transmitter-with-waterproof-cable-sensor-id-a4-30-3905/)| Temperature | Humidity | Temperature<BR> probe | 1 | No |
| [A5](https://www.tfa-dostmann.de/en/product/temperature-transmitter-id-a5-30-3901/) | Temperature | - | - | 5 | No |
| [A5](https://www.tfa-dostmann.de/en/product/pool-transmitter-tfa-me-id-a5-pool-30-3907/) | Temperature (Pool) | - | - | 5 | No |
| [A6](https://www.tfa-dostmann.de/en/product/professional-temperature-humidity-transmitter-id-a6-30-3906/)| Temperature | Humidity | - | 1 | No |

(*) TX: Transmission interval in minutes

All sensors additional deliver a low battery value.

{% enddetails %}

{% tip %}
{% endtip %}

## Station: Basic configuration steps

**Quick start:**

{% details "1. Connect your station to your WiFi" %}

**Note**: You can skip this step when your station is already connected to your WiFi.
Otherwise please follow the instructions from the datasheets above. Basically it works like this:

1. Insert power supply cable and batteries
2. Press **WiFi button** for 5 seconds until you hear a **"beep"**. (The WiFi button placed in battery case)

> 3. Go to the WiFi settings of your device (Smartphone, tablet, PC) and look for a network named **"TFA-ME-ID-XXX-XXX-XXX"**. XXX-XXX-XXX is the 9 digit serial number of your station.

Reviewer: ```suggestion
3. Go to the Wi-Fi settings of your device (Smartphone, tablet, PC) and look for a network named **TFA-ME-ID-XXX-XXX-XXX**. XXX-XXX-XXX is the 9-digit serial number of your station.
```

> 4. Connect your device to WiFi network **"TFA-ME-ID-XXX-XXX-XXX"**

Reviewer: ```suggestion
4. Connect your device to the Wi-Fi network **TFA-ME-ID-XXX-XXX-XXX**
```

> 5. Open a browser and enter **"192.168.8.1"** or **"me.local"**

Reviewer: ```suggestion
5. Open a browser and enter 192.168.8.1 or me.local`.
```

> 6. Select your WiFi network from the list, enter your password and press **"Submit"**

Reviewer: ```suggestion
6. Select your Wi-Fi network from the list, enter your password, and select **Submit**
```
- we use "press" for [physical buttons](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/press-and-hold).
- we use just bold to markup UI strings, not "".

Author: OK, thank you!

> 7. Wait a few seconds until your station is connected to your WiFi network

Reviewer: ```suggestion
7. Wait a few seconds until your station is connected to your Wi-Fi network
```

{% enddetails %}

{% details "2. Activate Home Assistant funtionality of your station" %} 

> **Use the station's network settings:**

Reviewer: ```suggestion
**Use the station's network settings**:
```

---
title: TFA.me
description: Instructions on how to integrate the TFA.me integration into Home Assistant.

> ha_release: 2025.9

Reviewer: ```suggestion
ha_release: "2025.10"
```
To be defined when the core PR is merged

Author: Is this done by the maintainers or do I have to do change the release info?

Reviewer: the maintainer should check it in the end.

ha_iot_class: Local Poll
ha_codeowners:
  - '@DrMatthiasBlaschke'
ha_domain: a_tfa_me_1
ha_integration_type: hub
ha_category:
  - Sensor
ha_platforms:
  - diagnostics
  - sensor
works_with:
  - local

related:
  - url: https://developers.home-assistant.io/docs/documenting/standards
    title: Documentation standard
  - url: https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/
    title: Integration Quality Scale - Rules
  - docs: /docs/glossary/
    title: Glossary
  - docs: /docs/tools/quick-bar/#my-links

>     title: My link

Reviewer: ```suggestion
```
these are links from the template. You can use this section to add links that are relevant to your own integration. They will be rendered in a **Related topics** section at the bottom of the page.

---

> ## About TFA.me

Reviewer: ```suggestion
```

> The **TFA.me {% term integration %}** is used to integrate the **TFA.me** devices (stations & sensors) of [TFA Dostmann](https://www.tfa-dostmann.de/en/).

Reviewer: ```suggestion
The **TFA.me {% term integration %}** is used to integrate the **TFA.me** devices (weather and measuring instruments) of [TFA Dostmann](https://www.tfa-dostmann.de/en/).
```

Author: - Texts shortened

Author: Texts shortened

> 
> **TFA Dostmann** is an expert for weather and measuring instruments. With more than 1,000 weather and measuring instruments, TFA Dostmann GmbH & Co. KG is one of the leading companies in the industry in Europe. Since 1964, this family-owned company has been producing meteorological instruments at its headquarters in Wertheim am Main (Germany).
> 

Reviewer: ```suggestion
```
I think the first line is enough. For more information, people can view the TFA website.

> The **TFA.me system** offers a unique combination of professional, multi-day weather forecasts and precise monitoring of local measurements. Developed in Germany, this sophisticated system is characterised by a reliable WLAN connection and a user-friendly data display. The TFA.me system is the perfect choice for anyone who attaches great importance to up-to-date, reliable weather forecasts, but also wants to measure weather and indoor climate data using their own local sensors and use this online on a powerful platform.

Reviewer: ```suggestion
The **TFA.me system** offers multi-day weather forecasts and precise monitoring of local measurements.
```

Author: - Texts shortened

> ---

Reviewer: ```suggestion
```
We don't use these lines to mark sections.

## How it works

Each **TFA.me station** measures temperature and humidity internally, and depending on the model, other measurement data as well. Each station also has an 868 MHz radio receiver that can receive external {% term sensors %} such as temperature, humidity, wind and rain sensors. 

The internal and external measurement values ​​are partially displayed on the station's display and also transferred to the [**TFA.me cloud**](https://go.tfa.me). When measured values ​​are received station will add the current time stamp (UTC) and the 868 MHz RSSI (Radio Signal Strength Indicator) value to the data set. 

In addition, each **TFA.me station** also has the ability to hold the most recent measurement data in a table and make it available to Home Assistant via an integrated web server. Therefore, Home Assistant can query measurement data directly and locally from your station via the LAN (WiFi).

{% include integrations/config_flow.md %}

---

## Supported TFA.me devices

> Currently (summer 2025), the following [TFA.me stations and sensors ](https://www.tfa-dostmann.de/en/produkte/wetterstationen/wetterstationen-wlan/tfa-me/)are supported. This product range is continuously being expanded. {% details "Stations and sensors list ..." %}

Reviewer: ```suggestion
Currently (summer 2025), the following [TFA.me stations and sensors ](https://www.tfa-dostmann.de/en/produkte/wetterstationen/wetterstationen-wlan/tfa-me/) are supported. This product range is continuously being expanded. 

{% details "Stations and sensors list ..." %}
```

[TFA.me](https://www.tfa-dostmann.de/en/produkte/wetterstationen/wetterstationen-wlan/tfa-me/) Stations 

All stations deliver cyclic the following internal measurement values. 

> | ID | Value 1 | Value 2 | Value 3 | TX(*) | Events |

Reviewer: We [prefer lists instead of tables](https://developers.home-assistant.io/docs/documenting/general-style-guide#tables), see [example](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L105)

Author: What is wrong with tables? Are there any technical or other problems with the display?
Tables offer the advantage over lists in that the lines and headings make them much clearer than simply a flow text as part of a list entry. They offer a good graphical breakdown of table entries.
Therefore we would prefer tables.
Question: Is this just a recommendation/suggestion or a requirement?

Reviewer: It's part of the Style Guide, as indicated in the link provided in my earlier comment.

Author: Without the three tables, a meaningful display is impossible. Tables absolutely have their uses and are de facto used on most websites, including mobile devices. A table is a matrix. You can't represent the same thing with a row-based list.

Furthermore, the pages have been tested. There are no display issues on, for example, an iPhone. 
Also other companies integration help pages also use tables: e.g. "sonos", "zwave_js"

The style guide also says" ...If you cannot avoid a table, minimize the number of columns and keep the amount of text as short as possible..." And that exactly was done. Less columns, short texts.

Author: Changed all tables to list, even though I find this less clear.

- Station ID 03: [Quick start](https://com-tradebyte-core-tbone-media-live.s3.eu-central-1.amazonaws.com/media/1768/32486-da31350748054.pdf) or [manual](https://com-tradebyte-core-tbone-media-live.s3.eu-central-1.amazonaws.com/media/1768/32420-99cd613999497.pdf)
- Station ID 04: [Quick start](https://com-tradebyte-core-tbone-media-live.s3.eu-central-1.amazonaws.com/media/1768/32492-a085903128848.pdf) or [manual](https://com-tradebyte-core-tbone-media-live.s3.eu-central-1.amazonaws.com/media/1768/32432-3e4d128334262.pdf)
- Station ID 05: [Quick start](https://com-tradebyte-core-tbone-media-live.s3.eu-central-1.amazonaws.com/media/1768/34198-944a947928662.pdf) or manual (coming soon)
- Station ID 06: [Quick start](https://com-tradebyte-core-tbone-media-live.s3.eu-central-1.amazonaws.com/media/1768/34202-71a5856072875.pdf) or manual (coming soon)
- Station ID 07: [Quick start](https://com-tradebyte-core-tbone-media-live.s3.eu-central-1.amazonaws.com/media/1768/34200-288c483905760.pdf) or manual (coming soon)
- Station ID 08: [Quick start](https://com-tradebyte-core-tbone-media-live.s3.eu-central-1.amazonaws.com/media/1768/34196-acf3991570949.pdf) or manual (coming soon)

---

> ## Entities

Reviewer: This section should be called [supported functionality](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L88)

Author: OK, done.

| wind_direction | SE, SSE| {% icon "mdi:arrow-top-left" %}  | 6, 7 |
| wind_direction | S, SSW | {% icon "mdi:arrow-up" %}          | 8, 9 |
| wind_direction | SW, SSW| {% icon "mdi:arrow-top-right" %}   | 10, 11 |
| wind_direction | W, WNW | {% icon "mdi:arrow-right" %}          | 12, 13 |
| wind_direction | NW, NNW| {% icon "mdi:arrow-bottom-right" %}      | 14, 15 |
| wind_speed     | default   | {% icon "mdi:weather-windy-variant" %} | | 
| wind_gust      | default   | {% icon "mdi:weather-windy" %} | |
| rain | none     | {% icon "mdi:weather-sunny" %} | (< 0.1 mm)/hour|
| rain | light    | {% icon "mdi:weather-partly-rainy" %} | (0.1 mm ... < 0.5 mm)/hour|
| rain | moderate | {% icon "mdi:weather-rainy" %} | (0.5 mm ... < 4.0 mm)/hour|
| rain | heavy    | {% icon "mdi:weather-pouring" %} | (> 4.0 mm)/hour|

{% enddetails %}
{% enddetails %}

--- 
 
## Removing the integration

This integration follows the standard integration removal process; no extra steps are required.

{% include integrations/remove_device_service.md %}

Optional: If you no longer use the Home Assistant functionality in your station, you can also deactivate it again.

---

## Troubleshooting

> If you have any problems with please join the [Home Assistant Help](https://www.home-assistant.io/help/) pages.

Reviewer: This is very generic advice. Do you have any integration-specific troubleshooting steps? If not, you can also add a troubleshoooting section later, as issues come up.

Author: I understand what you mean, but we have to wait until the first customers use the integration and send feedback. Then we can collect issues and, if necessary, set up a dedicated help/troubleshooting page. Therefore, for now, only the very general reference to the Home Assistance help pages. So at the moment we only  have the instructions and link/URLs to "quick start" and/or "manual" PDFs for all weather station and sensors above.

Reviewer: ~yes, exactly. I would remove this section and only add it once issues come up.~ On the other hand, linking to the help page is better than nothing.

Author: OK, let's leave it in for now until something better comes along.
